workflows/codefresh-csdp/versions/1.1.4/images/image-enricher-jira-info/src/jira.service.js

```diff
@@ -1,16 +1,20 @@
 const JiraClient = require('jira-connector');
+const { Version2Client } = require('jira.js');
+
 const _ = require('lodash');
 
 const inputs = require('./configuration').inputs;
 const codefreshApi = require('./codefresh.api');
 const CodefreshJiraClient = require('./CodefreshJiraClient');
 
 class JiraService {
+    // jira: Version2Client
 
     async init() {
         let jiraConfig = _.cloneDeep(inputs.jira)
 
         if (jiraConfig.context) {
+            console.log('Using jira context auth')
```

Comment:
let's change new logs to start with lowercase latter because it's like that in other places